crates/lib3h/tests/test_suites/two_basic.rs

```diff
@@ -151,16 +152,40 @@ pub fn test_send_message(alex: &mut NodeMock, billy: &mut NodeMock) {
 /// Test SendDirectMessage and response
 #[allow(dead_code)]
 fn test_send_message_fail(alex: &mut NodeMock, _billy: &mut NodeMock) {
-    // Send to self
-    let _req_id = alex.send_direct_message(&ALEX_AGENT_ID, "wah".as_bytes().to_vec());
-    let expected = "FailureResult\\(GenericResultData \\{ request_id: \"req_alex_3\", space_address: HashString\\(\"appA\"\\), to_agent_id: HashString\\(\"alex\"\\), result_info: ";
-    assert_msg_matches!(alex, expected);
-
     trace!("[test_send_message_fail] alex send to camille");
     // Send to unknown
     let _req_id = alex.send_direct_message(&CAMILLE_AGENT_ID, "wah".as_bytes().to_vec());
 
-    let expected = "FailureResult\\(GenericResultData \\{ request_id: \"req_alex_4\", space_address: HashString\\(\"appA\"\\), to_agent_id: HashString\\(\"camille\"\\), result_info: ";
+    let expected = "FailureResult\\(GenericResultData \\{ request_id: \"req_alex_3\", space_address: HashString\\(\"appA\"\\), to_agent_id: HashString\\(\"camille\"\\), result_info: ";
+    assert_msg_matches!(alex, expected);
+}
+
+/// Test SendDirectMessage and response to self
+pub fn test_send_message_self(alex: &mut NodeMock, _billy: &mut NodeMock) {
```

Comment:
will this break the websocket version of these tests? I guess when we un-ignore them, we can deal with skipping this one?